rlclientlib/live_model.cc

```diff
@@ -49,6 +49,15 @@ namespace reinforcement_learning
     return err_code;
   }
 
+  std::vector<int> live_model::c_array_to_vector(const int* c_array, size_t array_size)
+  {
+    if (c_array == nullptr)
+    {
+      return live_model::default_baseline_vector;
```

Comment:
it is weird to have method with generic name ("c_array_to_vector"), which actually returns default_baseline_vector (which is about baselines => more specific than arbitrary arrays). So I would replace reference to default_baseline_vector with just explicit vector() here.

Reply:
agreed